DDPG:
  alfa_actor: 0.1
  alfa_critic: 0.1
  discount_factor: 0.99
  soft_target_update: 0.005
  
ADAM_param:
  actor: 0.0001 #0.0003
  critic: 0.0001 #0.0003
  
Ornstein_Uhlenbeck:
  teta: 0.5 #0.15
  sigma: 1.0 #0.2 #0.3 #0.2  
  mu: 0

Spatial_Attention :
  #spatial_att_ourdim : 64 #spatial att output dim second version
  spatial_att_outdim : 30 # spatial attention output dimension
  n_sector_spatialatt : 10 # number of elements inside each section of laser scan
  input_spatial_size : 10 # number of section used inside spatial attention
  n_rays : 100
  #embedding_output_size : 128 #output dim of embedd net inside spatial dim second version
  embedding_output_size : 64 #output dimension of embedding network inside spatial attention 

ICP:
  n_iter : 100
  threshold : 1.0  
  error_thresh : 0.001

TAGD:
  d_thresh : 0.25
  Nc : 15

Temporal_Attention:
  temporal_att_outdim : 30


Training :
  nepisodes  : 1000
  nsteps : 150
  buffer_size: 500000
  batch_size: 64
  n_waypoint : 5
  ahead_dist : 0.6 # Global path point ahead respect the current 
  single_goal : True
  dyn_path : False
  goal_pos : True
  test : False
  attention_module_flag : True
  scan_type : 'cartesian'
  max_ray_value : 3.5
  dist_waypoint : 0.3 #0.3
  algorithm : 'DDPG' #Possible algortihm ( DDPG , TD3 , PPO , SAC )
  debug : False
  #permit to save data used during debug phase 
  #batch_size : 128


  
